source/style-guide/style/lists.txt

```diff
@@ -39,8 +39,11 @@ guidelines when introducing lists.
        period instead of a colon.
 
        .. note::
-          Avoid using fragments to introduce lists. Fragments can be
-          harder to understand than sentences.
+
+          Where possible, avoid using fragments to introduce lists. 
+          However, you may use stem sentences to avoid especially 
+          awkward or redundant constructions. Use your best discretion. 
```

Comment:
Suggestion:

I think it would be helpful to continue to inform the reader why fragments should be avoided ("harder to understand than sentences"). I think the following phrasing might be more concise without losing any essential info:

"Avoid using fragments to introduce lists since they can be more difficult to understand than sentences. In cases in which sentences make the introduction awkward or redundant, you can use sentences stems."